src/usdb_syncer/settings.py

```diff
@@ -122,6 +122,34 @@ def default() -> Newline:
         return Newline.LF
 
 
+class CoverMaxSize(Enum):
+    """Maximum cover size."""
+
+    PX_3840 = 3840
+    PX_1920 = 1920
+    PX_1200 = 1200
+    PX_1000 = 1000
+    PX_640 = 640
+    PX_500 = 500
```

Comment:
Didn't you say 3 options would be enough? What changed your mind?
Also, it was possible to disable scaling completely before. Might be a viable alternative to the 3840px option.

Reply:
My reasoning was this:
1920x1920 is the max cover size for USDX on 4K screens. Consequently, 3940x3940 is the maximum needed by an 8K screen. 1200x1200 is the standard cover size provided by Apple, 1000x1000 is the standard cover size of fanart.tv, 640x640 is the standard cover size provided by spotify. 500x500 is for people who really try to save some disk space.

I agree that maybe we can change the 3940x3940 case to a disable option. What other option(s) would you get rid of?

Reply:
I guess 1200 and either 500 or 640. Fanarts sounds like a reasonable benchmark, but the other ones probably aren't very relevant for the syncer, are they? Especially since users won't know the origin of these options anyway.